src/category_theory/opposites.lean

```diff
@@ -183,6 +206,50 @@ end
 
 end functor
 
+namespace nat_trans
+
+variables {D : Sort u₂} [𝒟 : category.{v₂} D]
+include 𝒟
+
+section
+variables {F G : C ⥤ D}
+
+protected definition op (α : F ⟶ G) : G.op ⟶ F.op :=
+{ app         := λ X, (α.app (unop X)).op,
+  naturality' := begin tidy, erw α.naturality, refl, end }
+
+@[simp] lemma op_app (α : F ⟶ G) (X) : (nat_trans.op α).app X = (α.app (unop X)).op := rfl
+
+protected definition unop (α : F.op ⟶ G.op) : G ⟶ F :=
```

Comment:
Is this the best name, given that we will (eventually) also want the operation `(α : F ⟶ G) : G.unop ⟶ F.unop` on natural transformations? (same with `right_op` below).

On second thought, probably this is the most used definition. Most of the time you have a functor `Cᵒᵖ ⥤ Dᵒᵖ` it will be of the form `F.op`.

Reply:
I'll mark this as resolved; I'm happy to change the names if you have a suggestion.